lib/matplotlib/axis.py

```diff
@@ -1610,8 +1610,17 @@ def _update_axisinfo(self):
         self.set_default_intervals()
 
     def have_units(self):
+        """
+        Return `True` if units or a converter have been set.
+        """
         return self.converter is not None or self.units is not None
 
+    def have_units_and_converter(self):
+        """
```

Comment:
Should we make this private so we can more easily change the future api?  Units are likely to get an overhaul with the new NASA Roses funding so I would be conservative about adding new units API just now.

Reply:
👍 done